bundles/org.openhab.binding.miele/src/main/resources/OH-INF/thing/coffeemachine.xml

```diff
@@ -28,6 +28,16 @@
 				<description>The identifier identifies one certain appliance on the ZigBee network.</description>
 				<required>true</required>
 			</parameter>
+			<parameter name="protocol" type="text">
+				<label>Protocol</label>
+				<description>The protocol used to communicate with the appliance</description>
+				<required>false</required>
+				<default>"ZigBee"</default>
```

Comment:
```suggestion
				<default>ZigBee</default>
```
Also for all other files.

Reply:
@kaikreuzer Ok, this means there is a problem with the default formatter in Eclipse. I did a Cmd-Shift-F in each file

Reply:
The formatter does not add any quotes. I guess that was simply a copy&paste error from you.